static/js/components/inputs/CountrySelectField.js

```diff
@@ -0,0 +1,41 @@
+import React from 'react';
+import _ from 'lodash';
+import iso3166 from 'iso-3166-2';
+import SelectField from './SelectField';
+
+let countryOptions = Object.keys(iso3166.data).map(code => ({
+  value: code,
+  label: iso3166.data[code]['name']
+}));
+countryOptions = _.sortBy(countryOptions, 'label');
```

Comment:
any particular reason this is outside of the class declaration? I would have probably declared this in a `render` method as a little helper, probably fine either way though.